src/server/game/Entities/Player/Player.cpp

```diff
@@ -3023,20 +3032,22 @@ void Player::SetInWater(bool apply)
 
 bool Player::IsInAreaTriggerRadius(const AreaTrigger* trigger) const
 {
+    static const float delta = 5.0f;
+
     if (!trigger || GetMapId() != trigger->map)
         return false;
 
-    if (trigger->radius > 0.f)
+    if (trigger->radius > 0)
     {
         // if we have radius check it
         float dist = GetDistance(trigger->x, trigger->y, trigger->z);
-        if (dist > trigger->radius)
+        if (dist > trigger->radius + delta)
```

Comment:
what the fuck is that???

so to fix it, "they" added artificial 5 meters???????? this is the "correct solution" ?? this is as dirty as before